src/scripts/main.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const form = document.querySelector('.contact-us__form');
+
+form.addEventListener('submit', (e) => {
```

Comment:
Уникай однобуквених назв змінних
```suggestion
form.addEventListener('submit', (event) => {
```